tests/test_models.py

```diff
@@ -1,38 +1,186 @@
+        self.assertEqual(wishlist.wishlist_id, None)
+        self.assertEqual(wishlist.customer_id, fake_wishlist.customer_id)
+        self.assertEqual(wishlist.wishlist_name, fake_wishlist.wishlist_name)
+        self.assertEqual(wishlist.created_date, fake_wishlist.created_date)
+
+    def test_add_a_wishlist(self):
+        """It should Create a Wishlist and add it to the database"""
+        wishlists = Wishlist.all()
+        self.assertEqual(wishlists, [])
+        wishlist = WishlistFactory()
+        wishlist.create()
+        # Assert that it was assigned an id and shows up in the database
+        self.assertIsNotNone(wishlist.wishlist_id)
+        wishlists = Wishlist.all()
+        self.assertEqual(len(wishlists), 1)
+
+    def test_read_wishlist(self):
+        """It should Read a Wishlist"""
+        wishlist = WishlistFactory()
+        wishlist.create()
+
+        # Read it back
+        found_wishlist = Wishlist.find(wishlist.wishlist_id)
+        self.assertEqual(found_wishlist.wishlist_id, wishlist.wishlist_id)
+        self.assertEqual(found_wishlist.customer_id, wishlist.customer_id)
+        self.assertEqual(found_wishlist.wishlist_name, wishlist.wishlist_name)
+        self.assertEqual(found_wishlist.created_date, wishlist.created_date)
+
+    def test_update_wishlist(self):
+        """It should Update a Wishlist"""
+        wishlist = WishlistFactory(wishlist_name="change name")
+        wishlist.create()
+        # Assert that it was assigned an id and shows up in the database
+        self.assertIsNotNone(wishlist.wishlist_id)
+        self.assertEqual(wishlist.wishlist_name, "change name")
+
+        # Fetch it back
+        wishlist = Wishlist.find(wishlist.wishlist_id)
+        wishlist.wishlist_name = "name is changed"
+        wishlist.update()
+
+        # Fetch it back again
+        wishlist = Wishlist.find(wishlist.wishlist_id)
+        self.assertEqual(wishlist.wishlist_name, "name is changed")
+
+    def test_delete_an_wishlist(self):
+        """It should Delete a Wishlist from the database"""
+        wishlists = Wishlist.all()
+        self.assertEqual(wishlists, [])
+        wishlist = WishlistFactory()
+        wishlist.create()
+        # Assert that it was assigned an id and shows up in the database
+        self.assertIsNotNone(wishlist.wishlist_id)
+        last_id = wishlist.wishlist_id
+        wishlists = Wishlist.all()
+        self.assertEqual(len(wishlists), 1)
+        wishlist = wishlists[0]
+        self.assertEqual(wishlist.wishlist_id, last_id)
+        wishlist.delete()
+        wishlists = Wishlist.all()
+        self.assertEqual(len(wishlists), 0)
+
+    def test_list_all_wishlists(self):
+        """It should List all Wishlists in the database"""
+        wishlists = Wishlist.all()
+        self.assertEqual(wishlists, [])
+        for wishlist in WishlistFactory.create_batch(5):
+            wishlist.create()
+        # Assert that there are not 5 wishlists in the database
```

Comment:
I think there is a typo in the comment.  I think you mean "Assert that there are _now_ 5 wishlists..."

Reply:
Will be fixed!